contracts/TradeFactory/TradeFactoryFeesHandler.sol

```diff
@@ -0,0 +1,73 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.4;
+
+import '@openzeppelin/contracts/token/ERC20/IERC20.sol';
+import '@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol';
+import '@openzeppelin/contracts/utils/structs/EnumerableSet.sol';
+
+import './TradeFactorySwapperHandler.sol';
+
+interface ITradeFactoryFeesHandler {
+  event SwapperFeeSet(address _strategy, uint256 _fee);
```

Comment:
```suggestion
  event SwapperFeeSet(address indexed _strategy, uint256 _fee);
```
Maybe ? Honestly, with the invention of the graph the indexed is kinda useless ... But who knows